RetrospectiveExtension.Backend/appsettings.json

```diff
@@ -11,5 +11,8 @@
         "Default": "Warning"
       }
     }
+  },
+  "ApplicationInsights":{
```

Comment:
Just for uniformity's sake - probably could have spaces between the colons and the subsequent characters